doc/widgets/wordenrichment.rst

```diff
@@ -26,9 +26,30 @@ Selected instances from corpus.
 Description
 -----------
 
-No description yet for this widget.
+**Word Enrichment** displays a list of words with higher p-values (higher significance) for a
+selected subset compared to the entire corpus. Lower p-value indicates a higher likelihood that
+the word is significant for the selected subset (not randomly occurring in a text). FDR (False
+Discovery Rate) is linked to p-value and reports on the expected percent of false predictions 
+in the set of predictions, meaning it account for false positives in list of low p-values.
+
+.. figure:: images/Word-Enrichment-stamped.png
+
+1. Information on the input.
+	- Cluster words are all the tokens from the corpus.
```

Comment:
Could we unify this with the names we used for inputs? We use Corpus here and Data above. Similarly selected subset here and selected data above.